src/cljs/athens/views/filesystem.cljs

```diff
@@ -217,32 +257,43 @@
         close-modal       (fn []
                             (when-not @loading
                               (dispatch [:modal/toggle])))
+        el (.. js/document (querySelector "#app"))
         remote-graph-conf (subscribe [:db/remote-graph-conf])
         db-filepath       (subscribe [:db/filepath])
         state             (r/atom {:input     ""
                                    :remote?   (:default? @remote-graph-conf)
                                    :tab-value 0})]
     (fn []
-      [:div (use-style modal-style)
-       [modal/modal
-        {:title    [:div.modal__title
-                    [:> LibraryBooks]
-                    [:h4 "Database"]
-                    (when-not @loading
-                      [button {:on-click close-modal} [:> Close]])]
-         :content  [:div (use-style modal-contents-style)
-                    [:> Tabs {:on-change (fn [_x y] (swap! state assoc :tab-value y))
-                              :value     (:tab-value @state)}
-                     [:> Tab {:label "Open Local"}]
-                     [:> Tab {:label "Create Local"}]
-                     [:> Tab {:label "Join Remote"}]]
-                    (cond
-                      (= 2 (:tab-value @state))
-                      [join-remote-comp remote-graph-conf]
-
-                      (= 1 (:tab-value @state))
-                      [create-new-local state]
-
-                      (= 0 (:tab-value @state))
-                      [open-local-comp loading db-filepath])]
-         :on-close close-modal}]])))
+      (js/ReactDOM.createPortal
+        (r/as-element [:div (use-style modal-style)
```

Comment:
Wait did we finally get React Portal to work? So we can now use this pattern for tooltips, dropdowns, etc.?

https://discord.com/channels/708122962422792194/800579670629679165/833918121224044573

Reply:
Created a PR for this https://github.com/shanberg/athens/pull/4/files